assets/localStorageHandler.js

```diff
@@ -0,0 +1,108 @@
+export function handleLocalStorage() {
```

Comment:
You are programming with "side effects" here.

The "localStorage" object is defined somewhere but not in this file or this function.
But you use and change it in this function.
This causes the object to be changed outside the "normal" flow of the program, this is called "side effect".

It would be better if this function has a parameter "localstorage" and then returns the changed object.

Reply:
Like this? I changed it but I am not sure if it is accordingly.